### PR TITLE
Clean up `isize`, `usize`. Don't bench allocation but formatting.

### PR DESCRIPTION
**Pull request by tbu-**
r? @alexcrichton 

Part of #20792 that wasn't done in your commit.

### CODE REVIEW

File: src/libcoretest/fmt/num.rs [hunk 1]
```diff
@@ -156,96 +156,98 @@ fn test_format_int_twos_complement() {
...
     use test::Bencher;
     use core::fmt::radix;
     use std::rand::{weak_rng, Rng};
+    use std::io::util::NullWriter;
 
     #[bench]
     fn format_bin(b: &mut Bencher) {
         let mut rng = weak_rng();
-        b.iter(|| { format!("{:b}", rng.gen::<uint>()); })
+        b.iter(|| { let _ = write!(&mut NullWriter, "{:b}", rng.gen::<u32>()); })
```
Comment (Gankro) [inline src/libcoretest/fmt/num.rs]: Is there a reason to feed this into a `let` and not just return it into the closure?

**Comment by alexcrichton** [comment]
r=me with @Gankro's comment

**Comment by heinrich5991** [comment]
@alexcrichton @Gankro's comments was addressed. r?

**Comment by tbu-** [comment]
@alexcrichton Now, Gankro's nits addressed and fixed the function names from `test_format_i32` to `test_format_int` and similar.

**Comment by alexcrichton** [comment]
@bors: r+ d909dad

Thanks!

**Pull request merged**